libraries/CoreAudio/CoreAudio.h

```diff
@@ -22,6 +22,8 @@
 #define BEEP_FREQUENCY 1000
 #define AUDIO_BLOCK 16
 
+#define ATT 0.5
```

Comment:
please define it as private `static constexpr float` and change the name to a full word for clarity